pkgs/tools/misc/yubico-piv-tool/default.nix

```diff
@@ -4,11 +4,11 @@
 
 stdenv.mkDerivation rec {
   pname = "yubico-piv-tool";
-  version = "2.2.1";
+  version = "2.3.0";
 
   src = fetchurl {
     url = "https://developers.yubico.com/yubico-piv-tool/Releases/yubico-piv-tool-${version}.tar.gz";
-    sha256 = "sha256-t+3k3cPW4x3mey4t3NMZsitAzC4Jc7mGbQUqdUSTsU4=";
+    hash = "sha256-oCoS2VRdHvehuZhgbYm3tlWl9aFDdzbPUdsIP4dvVak=";
   };
 
```

Comment:
```suggestion

  postPatch = ''
    substituteInPlace CMakeLists.txt --replace "-Werror" ""
  '';

```

Reply:
Thanks for the review.

Fixed the location of `postPatch` as you suggested.

But are you sure it should come after `src`?
Because `nix run github:jtojnar/nixpkgs-hammering yubico-piv-tool` told me to move the `postPatch` after `configureFlags`:

```
When evaluating attribute ‘yubico-piv-tool’:
warning: attribute-ordering
The lists of dependencies, including the attribute “nativeBuildInputs” should preferably come before attributes for overriding default phases’ “postPatch” attribute in the expression.

Near /home/aroussel/src/nixpkgs/pkgs/tools/misc/yubico-piv-tool/default.nix:14:3:
   |
14 |   postPatch = ''
   |   ^
Near /home/aroussel/src/nixpkgs/pkgs/tools/misc/yubico-piv-tool/default.nix:18:3:
   |
18 |   nativeBuildInputs = [ pkg-config cmake gengetopt ];
   |   ^
See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/attribute-ordering.md
```

?

Reply:
> But are you sure it should come after `src`?

Yes, I would recommend to run this check off.